BaseModel/src/com/wpi/cs509/teamA/ui/NodeManageMenu.java

```diff
@@ -0,0 +1,83 @@
+package com.wpi.cs509.teamA.ui;
+
+import java.awt.BorderLayout;
+import java.awt.FlowLayout;
+
+import javax.swing.JButton;
+import javax.swing.JDialog;
+import javax.swing.JPanel;
+import javax.swing.JPopupMenu;
+import javax.swing.border.EmptyBorder;
+import javax.swing.JLabel;
+import java.awt.Font;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JTextField;
+import java.awt.Dialog.ModalityType;
+import javax.swing.JMenuItem;
+
+/**
+ * This is the class that administrators uses to add nodes to database
+ * 
+ * @author CS 509-Team A
+ *
+ */
+
+@SuppressWarnings("serial")
+public class NodeManageMenu extends JPopupMenu implements ActionListener {
+	private int xPos;
+	private int yPos;
+	private ImageComponent imagePanel;
+	private JMenuItem mntmAdd;
+	private JMenuItem mntmDelete;
+	private JMenuItem mntmEdit;
+
+	private final String ADD = "Add Node";
+	private final String DELETE = "Delete Node";
+	private final String EDIT = "Edit Node";
```

Comment:
I think we should add static here to save memory... I will change this for you..